### PR TITLE
Fixes issue #450

### PR DESCRIPTION
**Pull request by Nazeeh21**
Add scrollbar CSS and fix issue #450

### CODE REVIEW

**Comment by Nazeeh21** [comment]
Don't know why the Vercel check is failing. Can I get the logs?

**Comment by idoshamun** [comment]
I need to approve it. Let me do it real quick

**Comment by Nazeeh21** [comment]
I think all checks have passed. 
Can you please review it and merge it if everything is alright?

File: packages/webapp/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "prettier": "@dailydotdev/prettier-config",
   "dependencies": {
     "@dailydotdev/shared": "*",
+    "caniuse-lite": "^1.0.30001239",
```
Comment (idoshamun) [inline packages/webapp/package.json:18]: Why did you add this dependency?

File: packages/shared/src/styles/base.css [hunk 2]
```diff
@@ -232,3 +232,25 @@ img.lazyload:not([src]) {
...
+  background: var(--theme-label-tertiary);
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: var(--theme-label-primary);
+}
+
+::-webkit-scrollbar-thumb:active {
+  background: var(--theme-label-primary);
+}
```
Comment (idoshamun) [inline packages/shared/src/styles/base.css:256]: Please add a new line at the end of the file.

**Review by idoshamun — CHANGES_REQUESTED** [review]
Thank you so much for your contribution.
I submitted two comments and I have another request to ask.
Please remove the `scrollbar` utility class in `utilities.css` and its code occurrences (I believe there's only one or so). It's no longer necessary.

File: packages/shared/src/styles/base.css (hunk 2 above)
Comment (Nazeeh21) [reply packages/shared/src/styles/base.css:256]: Yah, I'll add that

File: packages/webapp/package.json (hunk 1 above)
Comment (Nazeeh21) [reply packages/webapp/package.json:18]: I prompted from the terminal to run the command.
I will remove that.

**Comment by Nazeeh21** [comment]
I have made the requested changes. Please review it

File: packages/shared/src/styles/utilities.css [hunk 3]
```diff
@@ -16,30 +16,6 @@
   text-overflow: ellipsis;
 }
 
-.scrollbar {
```
Comment (idoshamun) [inline packages/shared/src/styles/utilities.css:19]: Please also remove the usage of this class throughout the code.

**Review by idoshamun — CHANGES_REQUESTED** [review]
I added one comment, please check it.
Also, I want to ask you to revert the changes to the `package-lock.json` as they are not necessary.
Thanks for bearing with me and fixing all my comments

**Comment by Nazeeh21** [comment]
Made the required changes as asked. Please check it 😅

**Review by idoshamun — APPROVED** [review]
Looks awesome! 🚀
Let's wait for the build to finish so I can do some visual testing and merge this thing 🙏
